### ISSUE TITLE
Layout List-Block in Editor broken in some Themes since WP 5.5

### ISSUE THREAD

**Issue by burnuser** [WordPress/gutenberg#24780]
**Describe the bug**
Since WP 5.5 the layout of the regular List-Block is broken in Gutenberg in some Themes (tested with Twenty Fourteen, Twenty Sixteen, Twenty Seventeen).
While the resulting webpage seems to be always OK, in Gutenberg editor the List-Block in the stated themes is shown on the left screen side, outside of the boundary of the regular editor content:

![WP55_LayoutListBlock_broken](https://user-images.githubusercontent.com/32030147/91149017-d7613900-e6ba-11ea-92b0-8fb56c0bdfa4.jpg)

It's OK in some other Themes (tested with Twenty Twelve, Twenty Nineteen, Twenty Twenty):
![WP55_LayoutListBlock_OK](https://user-images.githubusercontent.com/32030147/91149615-93baff00-e6bb-11ea-8cd0-a614b86a19f4.jpg)

**To reproduce**
1. Select one of the problematic themes (2014/2016/2017)
2. Use a regular List-Block in combination with other blocks (paragraph, ...)
3. See the problem

**Expected behavior**
WYSIWYG
(Like in WP 5.4 and before)

**Editor version (please complete the following information):**
- WordPress version: 5.5
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Tested with default and Gutenberg => both with same problem
- If the Gutenberg plugin is installed, which version is it? 8.8.0

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser: tested with Firefox + Chrome
 - Version last version

**Comment by youknowriad** [comment]
I wonder if this is some kind of conflict with editor styles. cc @kjellr @jasmussen @melchoyce

**Comment by jasmussen** [comment]
I can reproduce. Twenty Fourteen:

<img width="1213" alt="Screenshot 2020-08-27 at 08 43 03" src="https://user-images.githubusercontent.com/1204802/91403112-8c256280-e841-11ea-8da5-92a1bfefc574.png">

Many themes handle wide and fullwide by using "margin-left: auto; margin-right: auto;" — when I explicitly add that back, it looks like this, which I _think_ is correct:

![Screenshot 2020-08-27 at 08 43 28](https://user-images.githubusercontent.com/1204802/91403329-9d6e6f00-e841-11ea-905c-26ff6e1f06a5.png)

Diving in deeper, it looks like there's a generic `.wp-block` style in the Twenty Fourteen editor style that does exactly that:

<img width="1006" alt="Screenshot 2020-08-27 at 08 43 57" src="https://user-images.githubusercontent.com/1204802/91403554-ad864e80-e841-11ea-98cb-d2ef9fe82c18.png">

But said style is overwritten by other editor style CSS, in the case of the list block:

<img width="1063" alt="Screenshot 2020-08-27 at 08 44 16" src="https://user-images.githubusercontent.com/1204802/91403727-baa33d80-e841-11ea-97be-9392712136a2.png">

Specifically this from the very same editor style:

<img width="323" alt="Screenshot 2020-08-27 at 08 46 50" src="https://user-images.githubusercontent.com/1204802/91404240-e0c8dd80-e841-11ea-848d-f3c329e68544.png">

I'm not entirely sure why that worked in 5.4 and broke now, because it seems like that latter style would always take precedence. However it's possible that the general code cleanup and reduction in CSS specificity efforts we've done, are causing some side effects here.

_Ideally_ the solution would be for the Twenty Fourteen editor style, and others, to be rewritten to adress this.

**Comment by kjellr** [comment]
Interesting. I actually fixed this for Twenty Fifteen earlier this year: 

https://core.trac.wordpress.org/ticket/50029

At the time I didn't realize it was effecting other themes as well. As you noted @jasmussen the theme styles are interfering with this, so the best route would be to fix this in the themes themselves. I've created a new ticket here to track these fixes: 

https://core.trac.wordpress.org/ticket/51157
